packages/gasket-plugin-jest/generator/jest.config.js

```diff
@@ -7,6 +7,11 @@ const customJestConfig = {
   collectCoverageFrom: ['**/*.js'],
   testEnvironmentOptions: {
     url: 'http://localhost/'
+  },
+  // Parse css imports & style paths
+  moduleNameMapper: {
+    '^.+\\.(css)$': '<rootDir>/test/style-mock.js',
+    '@ux/pivot/styles': '<rootDir>/test/style-mock.js'
```

Comment:
Adding a style mock for css import is a good call.

However, the next mapper appears to be an app-specific import. This generated example jest config should not have it. If anything, add a comment or link to docs about customizing further.
```suggestion
```